vanilla-framework/vanilla-framework
# 1525: Remove display: inline-block from notification pattern

## Done

- Removed `display: block` from notification pattern, so it is a block by default therefore giving it full width of its container

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/notifications/notifications/
- Check that the notification block (not the paragraph text) takes up the full width of its container (the visible screen)

## Details

Fixes #1525
